addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/parts/ScEnvSshConnRowExpanded.js

```diff
@@ -88,25 +99,73 @@ class ScEnvSshConnRowExpanded extends React.Component {
         <Table.Cell colSpan="3" className="p3">
           <Grid columns={2} stackable>
             <Grid.Row stretched>
-              <Grid.Column width={12}>{this.renderInfo()}</Grid.Column>
+              {this.isAppStreamEnabled ? (
+                <Grid.Column width={12}>{this.renderAppStreamInfo()}</Grid.Column>
+              ) : (
+                <Grid.Column width={12}>{this.renderInfo()}</Grid.Column>
+              )}
               <Grid.Column width={4}>
                 <Segment className="flex items-center">
-                  <div className="w-100 overflow-hidden">{this.renderCountDown()}</div>
+                  <div className="w-100">{this.renderCountDown()}</div>
                 </Segment>
               </Grid.Column>
             </Grid.Row>
           </Grid>
-          <div className="mt3">
-            Example:
-            <Segment className="mt2">
-              {example} <CopyToClipboard text={example} className="ml2 mt0" />
-            </Segment>
-          </div>
+          {this.isAppStreamEnabled ? (
+            <></>
+          ) : (
+            <div className="mt3">
+              Example:
+              <Segment className="mt2">
+                {example} <CopyToClipboard text={example} className="ml2 mt0" />
+              </Segment>
+            </div>
+          )}
         </Table.Cell>
       </Table.Row>
     );
   }
 
+  renderAppStreamInfo() {
+    const interfaces = this.networkInterfaces;
+    const moreThanOne = _.size(interfaces) > 1;
+
+    return (
+      <div>
+        <b>Connection instructions for your AppStream workspace:</b>
+        <List bulleted>
+          <List.Item>
+            The IP Address or DNS of the instance.{' '}
+            {moreThanOne ? 'Ask your administrator if you are not sure which one to use:' : ''}
+            <List>
+              {_.map(interfaces, network => (
+                <List.Item key={network.value} className="flex">
+                  {this.renderHostLabel(network)}
+                  <CopyToClipboard text={network.value} />
+                </List.Item>
+              ))}
+            </List>
+          </List.Item>
+          <List.Item>Click the &apos;Connect&apos; button below to navigate to the AppStream instance</List.Item>
+          <List.Item>You downloaded the private key when you created the SSH key.</List.Item>
+          <List.Item>Save this key&apos;s content into AppStream&apos;s Notepad application in .PEM format</List.Item>
+          <List.Item>Open PuttyGen in AppStream and convert your private PEM key to PPK format</List.Item>
+          <List.Item>Enter the PPK file and private IP address details in Putty to connect to EC2</List.Item>
```

Comment:
Before this step shouldn't there be some step suggesting that the user needs to click `use this key` or something in SWB UI?

Reply:
These instructions appear after the user clicks on the 'Use this key' button. Since the 'Use this key' button is the only  component while selecting the key pair, I don't think guidance to click would be necessary.

Reply:
Hmm, that's a lot of work to do in one minute then. Can't we show these steps when the user clicks on connections? I thought if we decouple `use this key` button with instructions then the user has more time.

Reply:
Moving these instructions, as discussed.